File: week_02/src/main/java/Question13.java
/*
Autor  = Zaid Nayeb
Date   = 17/11/2025

13.	Month name & number of days using switch-case
Description: Input a month number (1–12) and display its name and number of days. Example:
Input:
2
Output:
February, 28 or 29 days
*/
public class Question13 {
    public static String monthInfo(int num){
        if (num < 1 || num > 12){
            throw new IllegalArgumentException("Invalid month");
        }
        String show = "";
        switch (num){
            case 1:
                show = "January, 31 days.";
                break;
            case 2:
                show = "February, 28 or 29 days.";
                break;
            case 3:
                show = "March, 31 days.";
                break;
            case 4:
                show = "April, 30 days.";
                break;
            case 5:
                show = "May, 31 days.";
                break;
            case 6:
                show = "June, 30 days.";
                break;
            case 7:
                show = "July, 31 days.";
                break;
            case 8:
                show = "August, 31 days.";
                break;
            case 9:
                show = "September, 30 days.";
                break;
            case 10:
                show = "October, 31 days.";
                break;
            case 11:
                show = "November, 30 days.";
                break;
            case 12:
                show = "December, 31 days.";
                break;


        }
        return show;
    }
}

File: week_03/src/Question_13.js
// Author : Zaid Nayeb
// Date   : 2025/11/28

/*
13.	Power of a number (without built-in function) Description: Compute a^b using repeated multiplication. 
Example: 
Input: 
2, 5 
Output: 
32 

*/
console.log(powerOfN(2,5))

function powerOfN(a,b){
    if (b === 0) {
        return 1;
    }
    let num = a;
    for (let i = 1; i < b; i++) {
        num *= a;
    }
    return num;
}

module.exports = powerOfN;